src/image-handler/index.js

```diff
@@ -162,8 +166,25 @@ module.exports = {
       const heightScale = heightOut ? heightOut/heightIn : widthOut/widthIn
       const widthScale = widthOut ? widthOut/widthIn : heightOut/heightIn 
 
+
+      let xPercent = 50
+      let yPercent = 50
+      if (allowedParams.x) xPercent = allowedParams.x ? Number.parseInt(allowedParams.x) : 50
+      if (allowedParams.y) yPercent = allowedParams.y ? Number.parseInt(allowedParams.y) : 50
```

Comment:
This is a super minor quibble but why not:

```javascript
      const xPercent = allowedParams.x ? Number.parseInt(allowedParams.x) : 50
      const yPercent = allowedParams.y ? Number.parseInt(allowedParams.y) : 50
```

Reply:
No good reason. I had something much different and then refactored it into this, but didn't notice I could simplify it further. I will update it.